src/Logger/LoggerFactory.php

```diff
@@ -10,9 +10,21 @@ class LoggerFactory
     /** @var Configuration */
     private $configuration;
 
+    /**
+     * LoggerFactory constructor.
+     *
+     * @param Configuration $configuration
+     *
+     * @throws \Exception if the timezone supplied in configuration is not recognised as a valid timezone
+     */
     public function __construct(Configuration $configuration)
```

Comment:
Could add unit test for new behavior?

Reply:
Test added https://github.com/lavary/crunz/pull/178/commits/5c06d5f443e04c9296d740bd99cd5c397e98c043